components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,6 +81,48 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server UI lets you:
+
+- List out cookbooks with the latest version
+- View different versions of a cookbook
```

Comment:
Try to keep lists parallel (The structure of  1 & 2 are very different)

https://developers.google.com/tech-writing/one/lists-and-tables#keep_list_items_parallel